Cargo.toml

```diff
@@ -14,6 +14,7 @@ members = [
   "eng/test/mock_transport",
   "sdk/storage",
   "sdk/storage/azure_storage_blob",
+  "sdk/temp-project/docs",
```

Comment:
I don't want this part of the main workspace. We should put this under `eng/tools` which can have a separate workspace, if even needed there.

Reply:
I see now. This is for a "template" project. We could use what @hallipr wants as well, which I plan to basically be Key Vault Secrets anyway, which is what most other languages use for their "template" projects.

Reply:
removed in favor of azure_template that got added last week